IdEmotion[] list_id_emotion